docs/stacks-guide-gcp.md

```diff
@@ -60,6 +64,7 @@ control over how the stack's installation is handled. Everything is
 Kubernetes object!
 
 ```
+kubectl create namespace gcp
 kubectl crossplane stack generate-install 'crossplane/stack-gcp:master' stack-gcp | kubectl apply --namespace gcp -f -
```

Comment:
Nothing to do about this now, but we will want to add `--cluster` here following the merge of https://github.com/crossplaneio/crossplane-cli/pull/11/files#diff-b2f3b00bde31c4b05bb3606c48765516R17

Reply:
Is that flag going to be included in 0.3?